pootle/apps/pootle_store/unit/timeline.py

```diff
@@ -219,6 +251,29 @@ def submissions(self):
                                        creation_time=self.object.commented_on)
                               .order_by("id"))
 
+    @cached_property
```

Comment:
im not sure there is much point using a cached_property if we dont do `list()` on the result